### PR TITLE
New version: orhun.git-cliff version 2.1.2

### PR DESCRIPTION
**Pull request by schembriaiden**
Checklist for Pull Requests
- [ ] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [ ] Is there a linked Issue?

Manifests
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [ ] This PR only modifies one (1) manifest
- [ ] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [ ] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/146093)

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
> Installer Verification Analysis Context Information: 
 Specified hash doesn't match.
 Actual hash:   868AD7FBAEB50ADF3330622DDD3D4CD478BD68EC65186D97385544485A3CDDA2
 Expected hash: 0fcebca56d0647644d34411f29f7ed166bf138da9ba07d8cf1f122ed983cc6fc
 Specified hash doesn't match.
 Actual hash:   07EA4618A6B8F12A21191C76C6C41E977EF79CE2CB0B411F6C8A67EC437CDF8E
 Expected hash: a10bb8ac7899167661d84cb365984e84026aa4f076b820ff335bca4ef13919f0
 Specified hash doesn't match.
 Actual hash:   9ABF430AD1F9A536EEC6107EE7F6D41E70578B6AD15C3CEC7BB2D10DB47ED698
 Expected hash: 46a112772f3e5608390fcd2b97a9a8f521cbb6f064bab35572e7ddc462169d00
 

(Automated response - build 872.)

File: manifests/o/orhun/git-cliff/2.1.2/orhun.git-cliff.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+PackageVersion: 2.1.2
+InstallerType: zip
+NestedInstallerType: portable
+UpgradeBehavior: install
+Installers:
+- Architecture: x86
+  NestedInstallerFiles:
+  - RelativeFilePath: git-cliff-2.1.2\git-cliff.exe
+  InstallerUrl: https://github.com/orhun/git-cliff/releases/download/v2.1.2/git-cliff-2.1.2-i686-pc-windows-msvc.zip
+  InstallerSha256: 0FCEBCA56D0647644D34411F29F7ED166BF138DA9BA07D8CF1F122ED983CC6FC
```
Comment (stephengillie) [inline manifests/o/orhun/git-cliff/2.1.2/orhun.git-cliff.installer.yaml:14]:
```suggestion

```

(Automated response - build 872.)

**Comment by vikingnope** [comment]
Tried this locally on a windows 11 vm, and it worked fine:

image

image

**Review by stephengillie — APPROVED** [review]

**Pull request merged**